packages/grpc-js/src/server.ts

```diff
@@ -87,6 +90,7 @@ import { CallEventTracker } from './transport';
 const UNLIMITED_CONNECTION_AGE_MS = ~(1 << 31);
 const KEEPALIVE_MAX_TIME_MS = ~(1 << 31);
 const KEEPALIVE_TIMEOUT_MS = 20000;
+const MAX_CONNECTION_IDLE_MS = 30 * 60 * 1e3; // 30 min
```

Comment:
According to [this document](https://grpc.io/docs/guides/keepalive/), the default value for the MAX_CONNECTION_IDLE option is INT_MAX, which means "unlimited".